patches/tModLoader/Terraria/Localization/Content/fr-FR/tModLoader.json

```diff
@@ -3,22 +3,22 @@
 		// Main Menu
 		"PatreonSupport": "Considerez de nous supporter financièrement: ",
 		"MenuModBrowser": "Navigateur",
-		// "MenuDownloadMods": "Download Mods",
-		// "MenuDownloadModsDescription": "The in-game Mod Browser to find, download, and update your Mods.",
+		// "MenuDownloadMods": "Télécharger des Mods",
+		// "MenuDownloadModsDescription": "Le Bavigateur de Mods dans le jeu pour trouver, télécharger, et mettre à jour votre Mods.",
 		"MenuMods": "Mods",
-		// "MenuManageMods": "Manage Mods",
-		// "MenuManageModsDescription": "Manage which Mods are active, delete Mods, and configure your Modpacks.",
-		"MenuModSources": "Mods source",
-		// "MenuDevelopMods": "Develop Mods",
-		// "MenuDevelopModsDescription": "Develop and publish Mods for others to use, using your Mod Source code",
+		// "MenuManageMods": "Gérer les Mods",
+		// "MenuManageModsDescription": "Gérez quels Mods sont actifs, supprimez les mods, et configurez votre Modpacks.",
```

Comment:
In "supprimez les mods", I believe it should be  "supprimez des Mods"
Also, modpacks is a plural so it hould be "vos Modpacks."